cristin-import/src/main/java/no/unit/nva/cristin/mapper/artisticproduction/MovingPictureExtractor.java

```diff
@@ -0,0 +1,44 @@
+package no.unit.nva.cristin.mapper.artisticproduction;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import no.unit.nva.model.instancetypes.artistic.film.MovingPictureSubtype;
+import no.unit.nva.model.instancetypes.artistic.film.MovingPictureSubtypeEnum;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+public interface MovingPictureExtractor {
+
+    String MINUTE = "MINUTT";
```

Comment:
This is also decalred public in `ArtisticProductionTimeUnit `

Reply:
good catch. Fixed